### PR TITLE
readd intake-astro

### PR DESCRIPTION
**Pull request by martindurant**
See https://github.com/conda-forge/staged-recipes/pull/7120

### CODE REVIEW

**Comment by martindurant** [comment]
@beckermr , given the linter message above, I am confuse. intake-astro-feedstock doesn't seem to exist as a repo or conda package, but is blocked here. I could rename it intake-astronomy, maybe?

**Comment by beckermr** [comment]
something happened with the permissions? Let's ping conda-forge core on the other PR and see what is up.

**Comment by beckermr** [comment]
OK Finger's crossed!

**Comment by martindurant** [comment]
I don't see an intake-astro feedstock :|

**Comment by beckermr** [comment]
OMG no!

**Comment by beckermr** [comment]
Looks like everything is backed up and it is not just you: https://github.com/conda-forge/staged-recipes/tree/master/recipes

**Comment by beckermr** [comment]
We can rekick travis and see what gives

**Comment by martindurant** [comment]
Yes please

**Comment by martindurant** [comment]
What do you think I should do from here to get https://github.com/conda-forge/intake-astro-feedstock to exist?

**Comment by beckermr** [comment]
Ohhh what I mean is that it should exist soon. We resolved an issue with staged recipes.

**Comment by martindurant** [comment]
OK, then fingers crossed again

**Comment by beckermr** [comment]
This commit should have taken care of it. https://github.com/conda-forge/staged-recipes/commit/2242fb58784eeeaca3da4d56f26d3592c94c7924

**Comment by beckermr** [comment]
Wait what?!

**Comment by beckermr** [comment]
Ok I am looking into this now.

**Comment by beckermr** [comment]
Wow ok. We def need @conda-forge/core to look at this. It appears that the recipe was deleted from master without the feedstock ever being created. There must be bad state somewhere. Can someone help us out here please?

**Comment by scopatz** [comment]
This can happen if we run out of API tokens.  It is not a big deal if the linter fails

**Comment by beckermr** [comment]
I am so confused. We are not worried about the linter. We merged a recipe into staged recipes and the feedstock is not being made. Are we talking about the same thing?

**Comment by scopatz** [comment]
Oh I thought the issue was that the linter was failing on the readd

**Comment by beckermr** [comment]
Right yeah. I think the test for if the repo exists is saying yes even though it doesn't? This must be why both feedstock creation and the linter are failing.

**Comment by beckermr** [comment]
IDK how this all works, hence me bringing in the big guns! ;p

**Comment by scopatz** [comment]
Basically, there is a travis CI job that runs on merges to master that runs a conda-smithy command to create a new repo. You can look at these CI jobs for the merge commits to see what happened

**Comment by beckermr** [comment]
So I did. In the CI the feedstock is only deleted, never made. Then another commit gets merged to delete it permanently.

**Comment by beckermr** [comment]
Let me ask this a different way. Why does the linter think the feedstock exists if no repo exists?

**Comment by scopatz** [comment]
It might have gotten added to the `conda-forge/feedstocks` repo?

File: recipes/intake-astro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.1" %}
+{% set name = "intake-astro" %}
+{% set sha256 = "8f2a5f49a10a52cd58770154075de27229242e9ae5c4e95c6a449b285653b90d" %}
+
+package:
+   name: intake-accumulo
```
Comment (ocefpaf) [inline recipes/intake-astro/meta.yaml]: I believe this is the problem. `intake-accumulo` does exists. The linter message is just reading another variable. (That is why I don't like the excess of jinja variables!)

**Comment by beckermr** [comment]
Mmk. How does the code in staged recipes decide if a feedstock exists?

**Comment by beckermr** [comment]
Or how does it decide to skip building some ?

**Comment by martindurant** [comment]
hahaha!
That's the wrong package name, should be intake-astro

File: recipes/intake-astro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python
+    - intake>=0.3
```
Comment (scopatz) [inline recipes/intake-astro/meta.yaml]: Also, doesn't this need a space? `intake >=0.3`

**Comment by martindurant** [comment]
so very sorry for all the hassle. Should I add a PR again?

File: recipes/intake-astro/meta.yaml (hunk 2 above)
Comment (martindurant) [reply recipes/intake-astro/meta.yaml]: added one, but I would have expected the linter to complain about things like that

Comment (ocefpaf) [reply recipes/intake-astro/meta.yaml]: Indeed that should not fail silently :-/

**Comment by ocefpaf** [comment]
> Should I add a PR again?

Yep. With only one `name` though :stuck_out_tongue_closed_eyes:

**Comment by beckermr** [comment]
Greasast!

**Comment by beckermr** [comment]
Thanks all!

**Comment by martindurant** [comment]
https://github.com/conda-forge/staged-recipes/pull/7772

**Pull request merged**